test/lib/testMayaUsdPythonImport.py

```diff
@@ -34,3 +36,30 @@ def testImportModule(self):
         # not causing USD libraries to be loaded any other way.
         stageCache = mayaUsdLib.StageCache.Get(True, True)
         self.assertEqual(type(stageCache).__name__, 'StageCache')
+
+    def testPixarModulesAreSafeForScripting(self):
+        """Test that a script node can import the
+           Pixar Python modules in a secure scripting
+           context"""
+
+        node = cmds.createNode("script")
+
+        # This Pixar import will fail and stop the script unless it was added to the secure import paths.
```

Comment:
The unit tests do not use the .mod files. All the variables are setup in the [ctest.cmake](https://github.com/Autodesk/maya-usd/blob/dev/cmake/test.cmake#L233) file. I think you need to modify that for the tests.

Reply:
Indeed, this is why I set MAYA_MODULE_PATH for the testMayaUsdPythonImport unit test [here](https://github.com/Autodesk/maya-usd/pull/3119/files#diff-9df95854a7077fa3a82f09331c026f5af41da0ca094280db79dece13749a3707R54). This is also where MAYA_SECURE_BATCH_ENABLE is added to stop the script from running if an unsecure module import is detected.